src/components/buttons/index.ts

```diff
@@ -6,6 +6,7 @@ import PauseButton from "./pause-button";
 import RetryButton from "./retry-button";
 import YesButton from "./yes-button";
 import PlayButton from "./play-button";
+import LevelBloonButton from './level-bloon-button';
```

Comment:
There is a spelling mistake for balloon

Reply:
The bloon is short for balloon word. It's a pun or word play.